automatic/openvpn/README.md

```diff
@@ -42,6 +42,7 @@ following package parameter can be set:
 * `/EasyRsa`          - install EasyRSA 3 scripts for X509 certificate management
 * `/DcoDriver`        - install the OpenVPN Data Channel Offload driver
 * `/TapDriver`        - install the TAP-Windows driver (NDIS6)
+* `/PLAP`             - install Pre-Logon Access-Provider
```

Comment:
Placement of the ```/PLAP``` parameter should be consistent between the ```README.md``` and the ```openvpn.nuspec``` file.  At present the order in ```README.md``` is TapDriver --> PLAP --> WintunDriver versus the order in ```openvpn.nuspec``` of WintunDriver --> PLAP --> Documentation.

Please ensure the placement of the ```PLAP``` parameter is consistent across the 2 files.

Given the updates to the install script I would suggest perhaps WintunDriver --> PLAP --> Documentation which would be consistent ordering across all files.